### PR TITLE
fix: Fix greeting message label in settings

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://linear.app/chatwoot/issue/CW-1565/add-the-context-on-welcome-message-settings

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

You can see it in inbox settings page

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by iamsivin** [comment]
[Ref](https://github.com/chatwoot/chatwoot/pull/7056#discussion_r1194473913)

File: app/javascript/dashboard/i18n/locale/en/inboxMgmt.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
         },
         "CHANNEL_GREETING_TOGGLE": {
           "LABEL": "Enable channel greeting",
-          "HELP_TEXT": "Automatically send a greeting message after the contact's first message in a conversation.",
+          "HELP_TEXT": "Instantly engage customers with automated greeting messages.",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]: This sentence is not clear enough to explain what this feature is about. I would re-consider and rewrite the sentence, even if it is wordy. This change feels more like a catchy heading.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please find the comment above.

File: app/javascript/dashboard/i18n/locale/en/inboxMgmt.json (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]: @Hricha-Shandily Can you take a look?

Comment (Hricha-Shandily) [inline app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]:
```suggestion
          "HELP_TEXT": "Automatically send greeting messages when customers start a conversation.",
```

Comment (Hricha-Shandily) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]: @pranavrajs @iamsivin Suggested a new line. Please see.

Comment (sojan-official) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]: @Hricha-Shandily  this ended up as the original copy :D.  But it wasn't very clear for users as the greeting message was not sent when someone clicked on the start conversation button.  

The greeting message is only sent once the user starts a conversation and then sends a message into the conversation.  We would have to capture that.

Comment (iamsivin) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]: CC: @pranavrajs @sojan-official

Comment (Hricha-Shandily) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:86]: > The greeting message is only sent once the user starts a conversation and then sends a message into the conversation. We would have to capture that.

"Auto-send greeting messages when customers start a conversation and send their first message."

**Comment by iamsivin** [comment]
@pranavrajs @sojan-official @Hricha-Shandily Added the commit in the PR.

**Review by Hricha-Shandily — APPROVED** [review]

**Pull request merged**